pkg/adapterManager/manager.go

```diff
@@ -149,6 +159,21 @@ func (m *Manager) Execute(ctx context.Context, cfgs []*config.Combined, attrs at
 		}
 	}
 
+	bags := make([]*attribute.MutableBag, numCfgs)
```

Comment:
Sucks that we have to alloc a temp structure for the bags. Is it worth updating `Merge` to take a single bag at a time and we handle the looping ourselves here? We don't need to alloc anything other than the channel, but we lose the chance to verify there are no conflicting keys before we start merging. However, aspects should declare their response attributes in their config, so I'd argue that guaranteeing no conflicts could be pushed to config validation.